File: HelloWorld/Java/Apresentacao.java
import java.util.Scanner;

public class Apresentacao {
    public static void main(String[] args) {
        String nome;
        int idade;
        int quant;
        double salario = 0;
        double altura;
        boolean maiorIdade;
    
        Scanner scanner = new Scanner(System.in);
        double maior = 0;

        System.out.println("\nDigite a quantidade de integrantes da família:");
        quant = scanner.nextInt();
        scanner.nextLine(); 

        for (int i = 0; i < quant; i++) {
            System.out.println("\nIntegrante " + (i + 1));

            System.out.println("\nDigite o nome:");
            nome = scanner.nextLine();

            System.out.println("\nDigite a idade:");
            idade = scanner.nextInt();

            System.out.println("\nDigite a altura:");
            altura = scanner.nextDouble();
            scanner.nextLine();

            if (altura > maior) {
                maior = altura;
            }

            if (idade >= 18) {
                maiorIdade = true;

                System.out.println("\nDigite o salário:");
                salario = scanner.nextDouble();
            } else {
                maiorIdade = false;
            }

            System.out.println("\n*** DADOS ***");
            System.out.println("Nome: " + nome);
            System.out.println("Idade: " + idade);
            System.out.println("Altura: " + altura);
            System.out.println("Maior de idade: " + maiorIdade);

            if (maiorIdade) {
                System.out.println("Salário: " +salario);
            }

            System.out.println("Pessoa mais alta até agora: " + maior);
        }

        scanner.close();
    }
}

File: HelloWorld/C-sharp/Apresentacao.cs
using System;
namespace HelloWorld.Base;

public class Apresentacao
{
    string nome;
    int idade;
    int quant;
    decimal salario;
    double altura;
    bool maiorIdade;

    public void Executa()
    {
        double maior = 0;

        Console.WriteLine("\nDigite a quantidade de integrantes da família:");
        quant = int.Parse(Console.ReadLine());

        for (int i = 0; i < quant; i++)
        {
            Console.WriteLine($"\nIntegrante {i + 1}");

            Console.WriteLine("\nDigite o nome:");
            nome = Console.ReadLine();

            Console.WriteLine("\nDigite a idade:");
            idade = int.Parse(Console.ReadLine());

            Console.WriteLine("\nDigite a altura:");
            altura = double.Parse(Console.ReadLine());

            if (altura > maior)
            {
                maior = altura;
            }

            if (idade >= 18)
            {
                maiorIdade = true;

                Console.WriteLine("\nDigite o salário:");
                salario = decimal.Parse(Console.ReadLine());
            }
            else
            {
                maiorIdade = false;
            }

            Console.WriteLine("\n*** DADOS ***");
            Console.WriteLine("Nome: " + nome);
            Console.WriteLine("Idade: " + idade);
            Console.WriteLine("Altura: " + altura);
            Console.WriteLine("Maior de idade: " + maiorIdade);

            if (maiorIdade)
            {
                Console.WriteLine("Salário: " + salario);
            }

            Console.WriteLine("Pessoa mais alta até agora: " + maior);
        }
    }
}